src/main/java/core/basesyntax/Buldoser.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Buldoser extends Machine {
```

Comment:
There is a typo in the class name 'Buldoser'. It should be 'Bulldozer' to correctly represent the machine it is modeling.